source/common/ssl/context_impl.cc

```diff
@@ -85,6 +85,30 @@ ContextImpl::ContextImpl(ContextManagerImpl& parent, Stats::Scope& scope,
     verify_mode = SSL_VERIFY_PEER;
   }
 
+  if (!config.certificateRevocationList().empty()) {
+    bssl::UniquePtr<BIO> bio(
+        BIO_new_mem_buf(const_cast<char*>(config.certificateRevocationList().data()),
+                        config.certificateRevocationList().size()));
+    RELEASE_ASSERT(bio != nullptr);
+
+    // Based on BoringSSL's X509_load_cert_crl_file().
+    bssl::UniquePtr<STACK_OF(X509_INFO)> list(
+        PEM_X509_INFO_read_bio(bio.get(), nullptr, nullptr, nullptr));
+    if (list == nullptr) {
+      throw EnvoyException(
+          fmt::format("Failed to load CRL from {}", config.certificateRevocationListPath()));
+    }
+
+    X509_STORE* store_ctx = SSL_CTX_get_cert_store(ctx_.get());
```

Comment:
Sorry, I know that I suggested it, but this should be actually `store`, not `store_ctx`.

Reply:
Also, I know that's not your code (and I cannot comment on it), but in the CA handling, there is a call to `SSL_CTX_get_cert_store()` on each iteration, instead of before the loop, like you're doing here.

Could you update the old code to match yours? Thanks!